src/app/bav/fields.js

```diff
@@ -4,19 +4,28 @@ module.exports = {
     journeyKey: "sortCode",
     validate: [
       "required",
+      {
+        type: "length",
+        fn: (value) =>
+          value.length == 6 || value.match(/^\D*\d{6}\D*$/)
```

Comment:
Why do we need `value.length == 6` ? Ruth said this, right? 

> If the user then enters 6 digits and it still includes a disallowed character e.g. 12345g
> Show: Enter a valid sort code. For example, 802650

Reply:
Yes so this is to ensure that string with 6 characters total like `12345f` trigger the `Enter a valid sort code. For example, 802650` error as do strings with 6 digits but additional disallowed characters like `123456h`

Reply:
ooh sorry yeah this is the success should have had my coffee first 👍